docs/current/742989-cookbook.md

```diff
@@ -625,6 +625,44 @@ Set the Hashicorp Vault URI, namespace, role and secret identifiers as host envi
 
 [Learn more](./guides/723462-use-secrets.md)
 
+
+## Error handling
+
+### Aborting with unhandled error
+
+The following code listing demonstrates how to abort the script with an unhandled error, which allows debugging with the stack trace of the crash during development.
+
+```go file=./cookbook/snippets/error-handling/unhandled/main.go
+```
+
+### Aborting gracefully
+
+The following code listing demonstrates how to abort the script without crashing.
```

Comment:
```suggestion
The following code listing demonstrates how to abort a Dagger pipeline gracefully.
```

Reply:
See if better?

Reply:
Changes the meaning. It's the program we want to abort (or terminate) cleanly and return to the shell (or whatever user interface), not the pipeline. When the pipeline fails, there's no more pipelines to run, so no other pipeline to abort (abort = don't run what comes next, just leave now).